pom.xml

```diff
@@ -60,7 +60,7 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.3.1</version>
+            <version>6.14.3</version>
```

Comment:
Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

Reply:
will revert